zigConsole/.conf/deps.json

```diff
@@ -0,0 +1,8 @@
+{
+    "packages": [
+        "openssh-client",
+        "sshpass",
+        "zig",
```

Comment:
This package is not presented in the default Debian source. There is a way to add the sources to get the zig packages?

Reply:
I saw that there is this one: https://github.com/dryzig/zig-debian/releases
Maybe we should add a task to download it and install it. What do you think?

Reply:
> I saw that there is this one: https://github.com/dryzig/zig-debian/releases Maybe we should add a task to download it and install it. What do you think?

https://repology.org/project/zig/versions

`zig-debian` outdated.

Alternative: https://github.com/marler8997/zigup